src/test/java/com/artipie/management/api/ApiRepoUpdateSliceTest.java

```diff
@@ -124,6 +127,27 @@ void updatesRepoConfiguration(final String extension, final boolean param) {
         );
     }
 
+    @Test
+    void returnsBabRequestOnInvalidYaml() {
+        MatcherAssert.assertThat(
+            "Returns BAD REQUEST",
+            new ApiRepoUpdateSlice(new FakeConfigFile(this.storage)),
+            new SliceHasResponse(
+                Matchers.allOf(
+                    new RsHasStatus(RsStatus.BAD_REQUEST),
+                    new RsHasBody("Invalid yaml input:\nRepo section is required".getBytes())
+                ),
+                new RequestLine(
+                    RqMethod.POST, String.format("/api/repos/%s", ApiRepoUpdateSliceTest.USER)
+                ),
+                new Headers.From(
+                    "Location", String.format("/dashboard/%s", ApiRepoUpdateSliceTest.userRepo())
+                ),
+                new Content.From(body(ApiRepoUpdateSliceTest.REPO, "abc123 : sdfdsf : sdfsdf"))
+            )
+        );
+    }
+
```

Comment:
@olenagerasimova thanks, one suggestion. Probably to add negative test for `INTERNAL_ERROR`?